### DISCUSSION TITLE
New/different input quantites/signals for EMA

### DISCUSSION THREAD

**Discussion by scoleli** [DLR-VSDC/IEEE-MVC-2023#35] [Q&A] [answered]
Hi,
is it allowed to use quantites not inside `vehiclebus` as input for the EMA?
Are we allowed to add some other outputs to the FMU model and use them in our EMA?

Elia

**Comment by rpintodecastro** [comment, marked as answer]
Hi Elia
We expect competitors to use quantities that are available in the vehiclebus. 
Other outputs from the FMU model shouldn't be used. This will allow us to have a fair evaluation and comparison between the different EMAs. 

Thanks 
Ricardo
